src/main/scala/Mathematics/FindMin.scala

```diff
@@ -0,0 +1,22 @@
+package Mathematics
+
+object FindMin {
+
+	/**
+	    * Method returns Minimum Element from the list
+	    *
+	    * @param listOfElements
+    	    * @return
+    	*/
+
+	def findMin(Elements : List[Int]): Int = {
```

Comment:
the same as for `findMax`: 
`def findMax(elements : List[Int]): Int = elements.foldLeft(elements.head){(acc, i) => if (acc > i) i else acc}`

Reply:
done